src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheckTest.java

```diff
@@ -83,7 +85,7 @@ public void testInvalidEdges()
         this.verifier.actual(this.setup.getInvalidEdges(),
                 new SinkIslandCheck(ConfigurationResolver.emptyConfiguration()));
         this.verifier.verifyExpectedSize(1);
-        this.verifier.verify(flag -> Assert.assertEquals(2, flag.getFlaggedObjects().size()));
+        this.verifier.verify(flag -> Assert.assertEquals(3, flag.getFlaggedObjects().size()));
```

Comment:
Why the change here?

Reply:
Previously, edge with aeroway tag was excluded from the check. But since aeroway tagged ways are not ingested as edges, that validation was removed and so this edge will be flagged as sink island.

Reply:
Ok. I'm a little confused about removing this validation -- was it creating false positives? Are we sure that the `line` definition won't change in the future?

Reply:
I think at some point aeroway ways were ingested as edges. Since these are not ingested as edges, the validation to remove aeroway ways from this check was not applied and so the edges with aeroways were still flagged as sink island. The new enhancement to exclude service roads within airports would capture these cases.